src/guides/v2.3/graphql/queries/url-resolver.md

```diff
@@ -47,13 +47,77 @@ The following query returns information about the URL containing `joust-duffle-b
 }
 ```
 
+### URL Resolver for the Category type
+
+The following query returns information about the URL containing `gear/bags.html` for the type Category.
+
+**Request:**
+
+```graphql
+{
+  urlResolver(url: "gear/bags.html") {
+    id
+    relative_url
+    redirectCode
+    type
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "urlResolver": {
+      "id": 4,
+      "relative_url": "gear/bags.html",
+      "redirectCode": 0,
+      "type": "CATEGORY"
+    }
+  }
+}
+```
+
+### URL Resolver for the Category type
```

Comment:
You have a duplicated title here - should it be, "URL resolver for CMS page" ?

Reply:
I will do changes.